src/clj/server/csv.clj

```diff
@@ -0,0 +1,37 @@
+(ns server.csv
+  (:require
+   [clojure.string :as string]
+   [shoreleave.middleware.rpc :refer [defremote]]))
+
+(defremote csv-files
+  []
+  (-> "./csv"
+      clojure.java.io/file
+      (remove #(.isDirectory %1))
+      (map #(.getName %1))
+      (filter #(.endsWith %1 ".csv"))))
+
+(defn- csv-name->fname
```

Comment:
I would prefer csv-name-to-fname. 

I don't like using -> in the function name since it's a macro(?) also, it kind of resembles a c++ class method call